code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer.dm

```diff
@@ -232,7 +232,7 @@
 		return
 	switch(action)
 		if("power")
-			toggle_power()
+			toggle_power(usr)
```

Comment:
no, bad 🗞️ 
if this is in ui_act the ui datum holds a reference to the user mob, use that. We do not guarantee that ui_act will be called off topic() forever

Reply:
What's the ui datum?

Reply:
![image](https://user-images.githubusercontent.com/58055496/231953609-9363ef11-bf4f-4250-aba1-a2e466b9c9ae.png)
/datum/tgui/ui here
it's not always explicitly marked in the args but it is always passed in

Reply:
Today i learn something new.